client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -82,15 +83,17 @@ const SiteMigrationInstructions: Step = function ( { navigation, flow } ) {
 	const queryParams = useQuery();
 	const fromUrl = queryParams.get( 'from' ) ?? '';
 
-	const { updateMigrationStatus } = useUpdateMigrationStatus();
+	const { mutate: updateMigrationStatus } = useUpdateMigrationStatus( siteId );
+
 	useEffect( () => {
 		if ( siteId ) {
-			updateMigrationStatus( siteId, 'migration-started-diy' );
+			updateMigrationStatus( { status: MigrationStatus.PENDING_DIY } );
```

Comment:
shouldn't this be behind the feature flag?

Reply:
I added the feature flag on the hook directly to avoid repeating the same checking.

https://github.com/Automattic/wp-calypso/pull/96103/files#diff-78935c1adabc2a1ba047481fbc0bf19fe577ba4b0cdc7fcfcaac98f8b24b0842R9

Reply:
Fixed here [bea3cbb](https://github.com/Automattic/wp-calypso/pull/96103/commits/bea3cbb570116012f865ff33e234878766cfcc62)